src/stories/3-fields/checkbox-group/checkbox-toggle-group.stories.tsx

```diff
@@ -0,0 +1,207 @@
+import { ArgsTable, Description, Heading, PRIMARY_STORY, Stories, Title } from "@storybook/addon-docs";
+import { Meta, Story } from "@storybook/react/types-6-0";
+import { ICheckboxGroupSchema } from "../../../components/fields/checkbox-group";
+import { CommonFieldStoryProps, FrontendEngine, SUBMIT_BUTTON_SCHEMA } from "../../common";
+
+export default {
+	title: "Field/Checkbox/Toggle",
+	parameters: {
+		docs: {
+			page: () => (
+				<>
+					<Title>Checkbox - Toggle</Title>
+					<Description>This component provides a set of toggles for user to select</Description>
+					<Heading>Props</Heading>
+					<ArgsTable story={PRIMARY_STORY} />
+					<Stories includePrimary={true} title="Examples" />
+				</>
+			),
+		},
+	},
+	argTypes: {
+		...CommonFieldStoryProps("checkbox"),
+		disabled: {
+			description: "Specifies if the checkbox should be disabled",
+			table: {
+				type: {
+					summary: "boolean",
+				},
+				defaultValue: { summary: false },
+			},
+			options: [true, false],
+			control: {
+				type: "boolean",
+			},
+		},
+		customOptions: {
+			description: "A custom options on which styling to use for rendering the toggle group.",
```

Comment:
- because the keys offered in customOptions can be ambiguous, there is a need to elaborate what each key does
- document the possible values of `styleType`